### ISSUE TITLE
error reporting for missing EnvironmentFile is hard to parse

### ISSUE THREAD

**Issue by strangemonad** [coreos/fleet#1121]
the error when trying to start a unit that refers to a missing EnvironmentFile doesn't provide much data:

Feb 08 23:31:25 core-01 systemd[1]: Failed to load environment files: No such file or directory
Feb 08 23:31:25 core-01 systemd[1]: ingestion.service failed to run 'start-pre' task: No such file or directory

It would be nice to at least print the file path that wasn't found

**Comment by bcwaldon** [comment]
This is a log line from `systemd` - note the `systemd[1]` token in the header of the log line.
